src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const th = document.createElement('th');
+
+th.textContent = table.tHead.rows[0].cells[1].textContent;
+table.tHead.rows[0].cells[3].after(th);
+table.tFoot.rows[0].cells[3].after(th.cloneNode(true));
```

Comment:
Ensure that the table has a `tHead` and `tFoot` with at least four cells in the first row. If not, this code will throw an error. Consider adding checks to ensure these elements exist before attempting to access them.